CONTRIBUTING.md

```diff
@@ -228,32 +229,40 @@ only pull requests targeted directly against master.
 - Tag the release (use `git tag -a`) and create a release in Github
 - Delete the `RC` branches
 
-### Point Release Procedure
+### Stable Release Update Procedure
 
 At the moment, only a single major release will be supported, so all point releases will be based
 off of that release.
 
 In order to alleviate the burden for a single person to have to cherry-pick and handle merge conflicts
-of all desired backporting PRs to a point release, we instead maintain a living backport branch, where
+of all desired backporting PRs to a point release, we instead maintain a living SRU branch, where
 all desired features and bug fixes are merged into as separate PRs.
 
 Example:
 
-Current release is `v0.38.4`. We then maintain a (living) branch `backport/release/v0.38.x`. As PRs
-are merged into master, if they wish to be backported into the next `v0.38.x` point release, the
-author must:
-
-1. Add `backport` label
-2. Pull latest changes on the desired `backport/release/v*.*.x` branch
-3. Create a 2nd PR merging the respective backport PR into `backport/release/v*.*.x`
-
-This means it is the authors responsibility to fix any merge conflicts, update changelog entries, and
-ensure CI passes. If a PR originates from an external contributor, it may be a team members responsibility
-to perform this process instead of the original author.
+Current release is `v0.38.4`. We then maintain a (living) branch `sru/release/v0.38.N`, given N as
```

Comment:
we should try to avoid saying what the current release. We will have to amend this every time we do a release.

Reply:
```suggestion
Given current release is `vMAJOR.MINOR.PATCH`. We then maintain a (living) branch `sru/release/vMAJOR.MINOR.PATCH+1`, given `N+1` as
```

Reply:
Does that work for you?

Reply:
@marbar3778 bump :)

Reply:
yea that works. thanks